Symfony/CS/Config/Config.php

```diff
@@ -53,6 +54,13 @@ public function setDir($dir)
         $this->dir = $dir;
     }
 
+    public function setCacheDir($cacheDir)
```

Comment:
@GrahamCampbell 

No problem, can add it.

However, I want to point this out:

* https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.6/Symfony/CS/Fixer.php#L128
* https://github.com/FriendsOfPHP/PHP-CS-Fixer/search?utf8=✓&q=usingCache

That is, `Config` has a method `usingCache()`, but it's not part of the `ConfigInterface` which is typehinted against in `Fixer::fix()`.